plugins/pulp_rpm/plugins/importers/yum/depsolve.py

```diff
@@ -163,9 +155,6 @@ def fills_requirement(self, unit):
         :return:        True if the unit satisfies the Requirement, False otherwise
         :rtype:         bool
         """
-        if self.name != unit.name:
```

Comment:
This is also problematic. If my requirement is ``kernel-devel-4.5.7-200``, ``kernel-4.5.7-200`` would also meet the requirement.